l10n_ch_dta/wizard/create_dta.py

```diff
@@ -724,10 +724,8 @@ def _create_dta(self, data):
                 record_type = RecordGt836
             elif country_code and country_code != 'CH':
                 record_type = RecordGt836
-            elif elec_pay == 'bvr':
+            elif elec_pay == 'postal':
```

Comment:
Here we should cover all type of bank with ccp. This also require to change the first test on IBAN as an IBAN can be a postal account (IBAN will be more widely used in the future).

Maybe we need an option on payment object to choose if we want to generate a BV or BVR which is the payment mode choice instead of a bank account type.